habitat-lab/habitat/config/habitat/task/rearrange/multi_agent_tidy_house_base.yaml

```diff
@@ -0,0 +1,61 @@
+# @package habitat.task
+
+defaults:
+  - /habitat/task: task_config_base
+  - /habitat/task/actions@actions.agent_0_base_velocity: base_velocity
+  - /habitat/task/actions@actions.agent_0_rearrange_stop: rearrange_stop
+  - /habitat/task/actions@actions.agent_0_pddl_apply_action: pddl_apply_action
+  - /habitat/task/actions@actions.agent_0_oracle_nav_action: oracle_nav_action
+  - /habitat/task/actions@actions.agent_1_base_velocity: base_velocity
+  - /habitat/task/actions@actions.agent_1_rearrange_stop: rearrange_stop
+  - /habitat/task/actions@actions.agent_1_pddl_apply_action: pddl_apply_action
+  - /habitat/task/actions@actions.agent_1_oracle_nav_action: oracle_nav_action
+  - /habitat/task/measurements:
+    - object_to_goal_distance
+    - articulated_agent_force
+    - force_terminate
+    - articulated_agent_colls
+    - composite_success
+    - num_steps
+  - /habitat/task/lab_sensors:
```

Comment:
For my understanding, which sensors go to which agents?

Reply:
I believe that here we specify which agents go to which agents:

https://github.com/facebookresearch/habitat-lab/blob/c991636450037455d165548598dbbd5dcc4d32ff/habitat-baselines/habitat_baselines/config/experiments_hab3/multi_agent_kinematic_oracle.yaml#L36